src/api/config/routes.rb

```diff
@@ -287,6 +287,12 @@ def self.public_or_about_path?(request)
       post 'project/remove_person/:project' => :remove_person, constraints: cons
       post 'project/remove_group/:project' => :remove_group, constraints: cons
       get 'project/monitor/:project' => :monitor, constraints: cons, as: 'project_monitor'
+      # For backward compatibility
+      get 'project/monitor', to: redirect { |_path_parameters, request|
+        url_string = request.query_parameters.except(:project).to_param
+        url_string = '?' << url_string unless url_string.empty?
+        "/project/monitor/#{request.query_parameters[:project]}?#{url_string}"
```

Comment:
There is still an extra`?` on this line. Please, remove it.

Reply:
My fault. Thanks. Done.